pom.xml

```diff
@@ -153,8 +153,10 @@
         <hive.archive.name>apache-hive-${hive.engine.hive.version}-bin.tar.gz</hive.archive.name>
         <hive.archive.mirror>${apache.archive.dist}/hive/hive-${hive.engine.hive.version}</hive.archive.mirror>
         <hive.archive.download.skip>false</hive.archive.download.skip>
+        <httpcore.version>4.4.15</httpcore.version>
         <httpclient.version>4.5.14</httpclient.version>
```

Comment:
Why do we reuse 4.4.15 for http core , based on the fact that we already use 4.4.16

Reply:
Just as based on the previous commit. Easy to be fixed before community agree with HC5 migration.